lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -12,8 +12,8 @@
 #include "cling/Interpreter/Interpreter.h"
 #include "cling/Interpreter/Transaction.h"
 #include "cling/Interpreter/Value.h"
-
 #include "cling/Utils/AST.h"
+#include "cling/Utils/Output.h"
```

Comment:
Sorry, I don't understand. Do you need this include, and if so, which declarations are used?